pjsip/include/pjsip/sip_parser.h

```diff
@@ -374,6 +374,7 @@ typedef struct pjsip_parser_const_t
 					     any chars except newlines or
 					     zero.			*/
     pj_cis_t pjsip_NOT_COMMA_OR_NEWLINE;/**< Array elements.		*/
+    pj_cis_t pjsip_NOT_SEMICOLON_OR_NEWLINE;/**< Array elements for privacy header.       */
```

Comment:
The commenting here is really ugly, but it was already like that before you changed this code. I’m happy for you to leave it alone, but tidying it up would be nice.